test/integration/dra/helpers_test.go

```diff
@@ -188,3 +188,36 @@ func waitForNotFound[T any](tCtx ktesting.TContext, get func(context.Context, st
 		return err
 	}).WithTimeout(60*time.Second).Should(gomega.MatchError(apierrors.IsNotFound, "IsNotFound"), "Object %T %s should have been removed.", t, name)
 }
+
+func waitForClaimAllocatedToDevice(tCtx ktesting.TContext, namespace, claimName string, timeout time.Duration) *resourceapi.ResourceClaim {
+	var latestClaim *resourceapi.ResourceClaim
```

Comment:
```suggestion
	tCtx.Helper()
	var latestClaim *resourceapi.ResourceClaim
```

Test failures should be attributed to the caller location, not the helper function.